apps/polyflix/package.json

```diff
@@ -16,6 +16,15 @@
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "@iconify/react": "^3.0.1",
+    "@milkdown/core": "^6.5.4",
+    "@milkdown/plugin-emoji": "^6.5.4",
+    "@milkdown/plugin-history": "^6.5.4",
+    "@milkdown/plugin-listener": "^6.5.4",
+    "@milkdown/preset-commonmark": "^6.5.4",
+    "@milkdown/prose": "^6.5.4",
+    "@milkdown/react": "^6.5.4",
+    "@milkdown/theme-nord": "^6.5.4",
+    "@milkdown/utils": "^6.5.4",
```

Comment:
I also noticed several bugs :

- Infinite loading when the note is empty : 
![image](https://user-images.githubusercontent.com/56176836/218873545-8b37ca46-ed52-4ea9-998d-5d0e9f0a764a.png)

- Weird UI with lines of code :
![image](https://user-images.githubusercontent.com/56176836/218874671-63e6e1bf-7280-496d-bb32-aea13ef6c5cd.png)

- Maybe you can add an indicator that the note is successfully saved because we don't know if it will be saved or not

Reply:
Also noticed that if you can't do a crtl+b and start writting in bold mode.
If you try to do it, you will only writte a single bold letter and next caracters will be vanilla.